lib/vtls/openssl.c

```diff
@@ -1520,7 +1520,8 @@ int cert_stuff(struct Curl_easy *data,
 
       PKCS12_PBE_add();
 
-      if(!PKCS12_parse(p12, key_passwd, &pri, &x509, &ca)) {
+      if(!PKCS12_parse(p12, key_passwd, &pri, &x509,
+                       &ca)) {
```

Comment:
```suggestion
      if(!PKCS12_parse(p12, key_passwd, &pri, &x509, &ca)) {
```
keep original line here? [nit]

Reply:
Ah, I just reverted the whole commit 3f79695. I can keep its formatting changes. Should I still leave git's default revert commit message untouched in this case?

Reply:
I think that's fine, if you make a separate sub-commit to amend it. However you feel it's best, it's also fine to leave it as-is.

Reply:
OK thanks! I did that first, but the double-revert looked strange to me and therefore I amended the commit message instead.